replay_analyser.py

```diff
@@ -99,6 +121,8 @@ def team_average_ratings(replays, cache, tank_info=None):
                 else:
                     teams[team_num].append((rating, tier))
 
+        if skipReplay:
+            break
```

Comment:
This _won't_ skip the replay but will stop any more replays being processed.
we want it to continue through, even though the result won't be accurate - this lets us at least get a list of unknown tanks for the run.